build.gradle

```diff
@@ -1,7 +1,11 @@
 plugins {
-    id "org.springframework.boot" version "1.5.9.RELEASE"
+    id 'org.springframework.boot' version '2.0.0.RELEASE'
```

Comment:
I didn't know you could combine the styles, thought they you had to do one or the other per https://plugins.gradle.org/plugin/org.springframework.boot .  Good to know

Reply:
Yeah, they rewrote the Gradle plugin and it reacts to the presence of other plugins (rather than drives).  Apparently idiomatic Gradle.